sdk/search/Azure.Search.Documents/src/Spatial/SpatialFormatter.cs

```diff
@@ -131,15 +131,19 @@ public static string EncodePolygon(GeographyLineStringProxy line)
 
             if (line.Points.Count < 4)
             {
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
```

Comment:
Should we just globally suppress this instead? As obvious below, we do this a lot and I don't feel we should stop doing it. It's more helpful information than saying, for example, that `nameof(line)` is invalid without saying exactly why. Or maybe we should include the property that is wrong in the message and not ignore these warnings.

Reply:
I don't think we should global suppress, the analyzer found real issue in other places.